src/opm/parser/eclipse/EclipseState/Aquifer/AquiferConfig.cpp

```diff
@@ -80,4 +80,12 @@ bool AquiferConfig::hasAquifer(const int aquID) const {
            numerical_aquifers.hasAquifer(aquID);
 }
 
+bool AquiferConfig::hasNumericalAquifer() const {
+    return this->numerical_aquifers.numAquifer();
+}
```

Comment:
I would prefer if this returned `std::size_t` and was called `size()`

Reply:
I will make it as follows, 

```
return (this->numerical_aquifers.size() > 0);
```